docs/blog/posts/2023-12-04-account-takeover.md

```diff
@@ -0,0 +1,144 @@
+---
+title: "Incident Report: User Account Takeover"
+description: A PyPI user had their account taken over
+author: Mike Fiedler
+publish_date: 2023-12-04 00:00
+date: "2023-12-04 00:00"
+tags:
+  - 2fa
+  - security
+  - transparency
+---
+
+# Incident Report: User Account Takeover
+
+## What happened?
+
+A PyPI user's account was taken over and used to remove the user's ownership of 4 projects.
+This was not a malfunction of PyPI or using any vulnerability,
+rather the user's account was not sufficiently protected against account takeover.
+
+The attacker added themselves as a collaborator to these projects,
+and removed the original owner. None of the projects had any modifications made to them other than ownership changes.
+
+After the attacker completed ownership changes, they deleted the user's account.
+
+PyPI Admins were informed by the user via email,
+blocked the attacker's access to the account,
+and restored the user's ownership of their projects.
+
+No further compromise was made, and **existing packages were not modified**.
+
+Read on for more details.
+
+## Timeline
+
+_all times are in UTC_
+
+System timings are from project ownership changes and email notifications.
+When a user's account is deleted, the account's data is removed immediately,
+thus we do not have timing of when the attacker gained access to `meisnate12`'s account.
+
+**Date: 2023-11-22**
+
+- 08:42:33 new user account `dvolk` created
+
+_(this is the first of 4 packages, only describing one here)_
+
+- 08:44:55 `meisnate12` account invites `dvolk` to be a collaborator on `arrapi`
+- 08:47:25 `dvolk` accepts the invitation
+- 08:47:35 `dvolk` removes `meisnate12` as a collaborator on `arrapi`
+
+_(repeated for `tmdbapis`, `nagerapi`, `pmmutils`)_
+
+- 08:50:27 `meisnate12` account is deleted
+
+No further actions by `dvolk` are observed after this point.
```

Comment:
```suggestion
No further actions by `dvolk` were observed after this point.
```

Reply:
Thanks - during incident reports, I tend to use "always now" tense, which doesn't make sense in a post-incident summary.